exampleApp/src/user/user.admin.ts

```diff
@@ -1,12 +1,31 @@
-import { AdminEntity, PasswordWidget } from "nestjs-admin";
-import { User } from "./user.entity";
+import { AdminEntity, DefaultAdminSite, PasswordWidget } from 'nestjs-admin'
+import { Request, Response } from 'express'
+import { User } from './user.entity'
+import { Injectable } from '@nestjs/common'
+import { Repository } from 'typeorm'
+import { InjectRepository } from '@nestjs/typeorm'
 
+@Injectable()
```

Comment:
Not sure making and entity Injectable is a good practice, would be happy to discuss this with you

Reply:
As seen together it isn't an entity in our database